app/src/main/java/com/zeapo/pwdstore/UserPreference.kt

```diff
@@ -174,7 +176,7 @@ class UserPreference : AppCompatActivity() {
                 true
             }
 
-            sshClearPassphrasePreference?.onPreferenceClickListener = ClickListener {
+            clearSavedPassPreference?.onPreferenceClickListener = ClickListener {
                 encryptedPreferences.edit().putString("ssh_key_local_passphrase", null).apply()
                 it.isVisible = false
```

Comment:
This seems off, are we only deleting one type of passphrase here? Also this should probably call `updateClearSavedPassphrasePrefs()` in the end.

Reply:
Fixed